services/example_service.md

```diff
@@ -1,12 +1,16 @@
 # Example Service
 
-Short description of the service and its purpose. This example showcases a service which contain: 
+This section should contain a short description of the service and its purpose. This example showcases a service which contain: 
```

Comment:
"If this was a real documentation and not and example"... This section should contain a.... "but it does not, because this is only an example or template."